### ISSUE TITLE
Set quick action background to white

### ISSUE THREAD

**Issue by aaronBery** [Legal-and-General/canopy#469]
Quick action text is not visible against certain backgrounds (doesn't meet accessibility standards) so we will update the background of the quick action to be white.
